### PR TITLE
Add Volume and Output Device listening

### PR DESCRIPTION
**Pull request by NathHorrigan**
This PR add's the native code and corresponding JS class for listening to audio output changes and volume.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -38,7 +38,31 @@ Sentry.init({
 export default function App() {
   const [loaded, setLoaded] = useState(false)
 
+  const notifyUserRegardingHeadphones = (connected: boolean) => {
+    if (connected == false) {
```
Comment (mixxorz) [inline src/App.tsx]:
```suggestion
    if (connected === false) {
```

File: src/screens/FearConditioningTrialScreen.tsx [hunk 2]
```diff
@@ -80,17 +86,32 @@ export const FearConditioningTrialScreen: ModuleScreen<FearConditioningTrialScre
   }
 
   useEffect(() => {
+    // Setup Volume Listening
+    soundSensorListener.current = AudioSensor.addVolumeListener(
+      (volume: number) => {
+        console.log('Volume Change!', volume)
```
Comment (mixxorz) [inline src/screens/FearConditioningTrialScreen.tsx]: Rogue console.log

**Review by mixxorz — COMMENTED** [review]
Looks good! Lots of Java/Swift I can't read but overall looks pretty good.

File: src/App.tsx (hunk 1 above)
Comment (NathHorrigan) [reply src/App.tsx]: Done

File: src/screens/FearConditioningTrialScreen.tsx (hunk 2 above)
Comment (NathHorrigan) [reply src/screens/FearConditioningTrialScreen.tsx]: Removed

**Pull request merged**